src/main/java/donnafin/logic/commands/RemoveCommand.java

```diff
@@ -27,7 +27,9 @@
             + ": removes a row from the attribute table. \n"
             + "Parameters: INDEX (must be a positive integer) "
             + "Example: remove 3";
-    private static final String MESSAGE_SUCCESS = "Successfully removed row.";
+    public static final String MESSAGE_SUCCESS = "Successfully removed row.";
+    public static final String MESSAGE_INVALID_TAB = "Successfully removed row.";
```

Comment:
is this duplicate intentional?

Reply:
jeez absolutely cancerous. thanks!
```suggestion
    public static final String MESSAGE_INVALID_TAB = "Wrong tab for RemoveCommand.";
```